pkg/daemon/daemon.go

```diff
@@ -234,6 +234,11 @@ func (d *daemon) AddPeriodicUpdate() {
 					glog.Errorf("AddPeriodicUpdate(): failed to update pod annotations with err: %v", err)
 					continue
 				}
+
+				if err = d.guidPool.ReleaseGUID(guidList[index].String()); err != nil {
+					glog.Warningf("AddPeriodicUpdate(): failed to release guid from removed pod with error: %v",
```

Comment:
please print guid and pod so it will be easy to debug

Reply:
added